app/models/user.rb

```diff
@@ -153,9 +153,7 @@ def pending_refund_payments
       projects: {
         state: 'failed'
       },
-      state: 'paid',
-      gateway: 'Pagarme',
```

Comment:
This was done since we can't refund payments from other gateways. I will keep this scope generic and update our request_refund method to use a bank account transfer instead of a refund for legacy gateways.